src/components/atoms/LatLonInputSet.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ChangeEvent, Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { MetAPIVariables } from '../../queries/metApi';
+import { TextInputField } from 'evergreen-ui';
+
+type LatLonInputSetProps = {
+    state: MetAPIVariables;
+    setState: Dispatch<SetStateAction<MetAPIVariables>>;
+};
+
+const LatLonInputSet: React.FC<LatLonInputSetProps> = (props) => {
+    const [state, setState] = useState<{ lat: string; lon: string }>({ lat: '63', lon: '11' });
```

Comment:
Add props to use pre-selected values in case the user navigated from the edit button in the Dashboard.

Reply:
This is part of #126 and #131